app/lib/paper_trail_version.rb

```diff
@@ -46,12 +46,12 @@ def shaped_changes
              end
   end
 
-  private
-
   def marked_urgent?
```

Comment:
so we define this function twice - once here and once in the audit_trail model.  IIRC the audit_trail is mongodb stuff? I see you commented out its tests below - should we comment out the file as well, or in some other way check that instance isn't being invoked?

audit_trail
https://github.com/DCAFEngineering/dcaf_case_management/blob/c5811041293e634bbdcb560311a9c464e3b749d1/app/models/audit_trail.rb#L24-L26

Existing Invocation
https://github.com/DCAFEngineering/dcaf_case_management/blob/c5811041293e634bbdcb560311a9c464e3b749d1/app/models/concerns/urgency.rb#L10

Reply:
yeah, the audit trail model is dead code. We have to keep it because builds get wonky without it, but it shouldn't actually contain anything that gets invoked anymore since we've switched all the models over as of the archivedpatient branch. Commenting out the contents of the class is a good idea though, so I'll do that!